evap/staff/templates/staff_user_bulk_delete.html

```diff
@@ -1,18 +1,22 @@
 {% extends "staff_base.html" %}
 
-{% load bootstrap3 %}
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="breadcrumb-item"><a href="{% url 'staff:user_index' %}">{% trans 'Users' %}</a></li>
+    <li class="breadcrumb-item">{% trans 'Bulk deletion' %}</li>
+{% endblock %}
 
 {% block content %}
     {{ block.super }}
-    <p>
-        {% trans "Upload a text file containing one username per line. All users that are not in this file which can be deleted will then be deleted." %}
-    </p>
+    <h2>{% trans 'Bulk delete users' %}</h2>
+
+    <p>{% trans "Upload a text file containing one username per line. All users that are not in this file will be deleted if possible. If they can't be deleted they will be marked inactive." %}</p>
```

Comment:
mixed quote style

Reply:
well, here we have the problem that the string contains an apostrophe which happens to be the same character as the single quote. maybe we should discuss this in more detail.